go.mod

```diff
@@ -1,6 +1,11 @@
 module github.com/juicedata/juicefs
 
-go 1.14
+go 1.16
+
+retract (
```

Comment:
This will break go 1.14
```
go: errors parsing go.mod:
/root/work/juicefs/go.mod:5: unknown block type: retract
Makefile:23: recipe for target 'juicefs' failed
make: *** [juicefs] Error 1
```

Reply:
Yep, Go < 1.16 doesn't recognize `retract` directive.